aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageHelper.java

```diff
@@ -172,6 +175,13 @@ public synchronized void update(
         }
     }
 
+    /**
+     * Deletes the SQLite database file from the device's local storage.
+     */
+    public void deleteDatabaseFromDisk() {
+        context.deleteDatabase(getDatabaseName());
```

Comment:
Should you call `super.close()` first, to be safe?

Reply:
I actually removed this code. I'm calling the helper's close method from the adapter where I can have better control to restart things